examples/chip-tool/main.cpp

```diff
@@ -115,14 +115,49 @@ static void HandleDataModelMessage(System::PacketBuffer * buffer)
     uint8_t * message;
     uint16_t messageLen = extractMessage(buffer->Start(), buffer->DataLength(), &message);
 
-    VerifyOrExit(messageLen == 5, printf("Unexpected response length: %d\n", messageLen));
+    VerifyOrExit(messageLen >= 3, printf("Unexpected response length: %d\n", messageLen));
     // Bit 3 of the frame control byte set means direction is server to client.
     // We expect no other bits to be set.
     VerifyOrExit(message[0] == 8, printf("Unexpected frame control byte: 0x%02x\n", message[0]));
     VerifyOrExit(message[1] == 1, printf("Unexpected sequence number: %d\n", message[1]));
-    VerifyOrExit(message[2] == 0x0b, printf("Unexpected command 0x%02x; expected Default Response", message[2]));
-    printf("Got default response to command '0x%02x' for cluster '0x%02x'.  Status is '0x%02x'.\n", message[3], frame.clusterId,
-           message[4]);
+
+    switch (message[2])
+    {
+    case 0x0b: {
```

Comment:
Magic number?

Reply:
It's the command id for the Default Response command.  Will document better.